components/molecules/FavoriteRepos/favorite-repos.tsx

```diff
@@ -0,0 +1,33 @@
+import Card from "components/atoms/Card/card";
+import Text from "components/atoms/Typography/text";
+import Title from "components/atoms/Typography/title";
+import FavoriteRepoCard, { FavoriteRepoCardProps } from "../FavoriteRepoCard/favorite-repo-card";
+import TableRepositoryName from "../TableRepositoryName/table-repository-name";
+
+interface FavoriteReposProps {
+  /** Limited to maximum of 3 items to show. */
+  repos: FavoriteRepoCardProps[];
+}
+
+const FavoriteRepos = ({repos}: FavoriteReposProps) => {
+
+  if(repos.length === 0) return null;
+  if(repos.length > 3) {
```

Comment:
remove this line and don't band-aid the number of returned repositories to 3 from the static component/client side (the api is set to take a custom limit)